test/doubly_linked_list.js

```diff
@@ -0,0 +1,183 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import DoublyLinkedList from '../src/doubly_linked_list'
+
+chai.use(chaiChange)
+
+describe('DoublyLinkedList', () => {
+  it('exists', function () {
+    expect(DoublyLinkedList).to.be.a('function')
+  })
+
+  context('next and previous pointers', () => {
```

Comment:
I am having a hard time understanding the purpose of this test based on this description.  Consider refining this test description to be more intention revealing.